csfieldguide/interactives/content/structure/interactives.yaml

```diff
@@ -166,6 +170,14 @@ jpeg-compression:
   languages:
     en: interactives/jpeg-compression.html
   is_interactive: true
+lzss-compression:
+  languages:
+    en: interactives/lzss-compression.html
+  is_interactive: true
+lzw-compression:
+  languages:
+    en: interactives/lzw-compression.html
+  is_interactive: true
```

Comment:
The ziv-lempel interactive isn't in this file

Reply:
sorry I'm not sure what you mean/what is wrong here?

Reply:
Perhaps I'm misunderstanding. It is in `content/en/interactives.yaml`, so I assume it exists, but not in this file - so it won't(?) be displayed on the `interactives` page nor translated